tutorials/maprapp/src/test/java/com/datatorrent/maprapp/ApplicationTest.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Put your copyright and license info here.
+ */
+package com.datatorrent.maprapp;
+
+import java.io.IOException;
+
+import javax.validation.ConstraintViolationException;
+
+import org.junit.Assert;
+
+import org.apache.hadoop.conf.Configuration;
+import org.junit.Test;
+
+import com.datatorrent.api.LocalMode;
+import com.datatorrent.maprapp.Application;
+
+/**
+ * Test the DAG declaration in local mode.
+ */
+public class ApplicationTest {
+  boolean check() {
+    return true;
+  }
+
+  @Test
+  public void testApplication() throws IOException, Exception {
+    try {
+      LocalMode lma = LocalMode.newInstance();
+      Configuration conf = new Configuration(false);
+      conf.addResource(this.getClass().getResourceAsStream("/META-INF/properties.xml"));
+      lma.prepareDAG(new Application(), conf);
+      LocalMode.Controller lc = lma.getController();
+      lc.runAsync();
+      while( !check() ) {
```

Comment:
This seems like it will exit immediately. And there is not anything being checked for the success of this test.
If this test is functional, then we should add some check; at least that the tuples are getting passed from kafka to hbase / maprdb.